packages/beacon-node/test/unit/chain/validation/attestation/validateAttestation.test.ts

```diff
@@ -0,0 +1,316 @@
+import {BitArray} from "@chainsafe/ssz";
```

Comment:
Spliced top level `describe` file from `attestation.test.ts` as the nature of tests requires different mocks.